app/src/org/commcare/android/javarosa/IntentCallout.java

```diff
@@ -303,10 +309,53 @@ private void storePointerToFileResponse(TreeReference ref, String responseValue,
         if (!src.exists()) {
             //TODO: How hard should we be failing here?
             Log.w(TAG, "CommCare received a link to a file at " + src.toString() + " to be included in the form, but it was not present on the phone!");
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/c0d09222e8b6b707b1789a28cfddc833d6f2a4c3/app/src/org/commcare/android/javarosa/IntentCallout.java#L311"></span>Using 'Log' instead of 'Timber'</td>
    </tr>
  </tbody>
</table>

Reply:
@ShivamPokhriyal I assume these are an autolinter of some sort? Am glad to get the inline feedback, although it seems like a few points (like this one) might need calibration. Should I be following their feedback?

Reply:
Correct, it's coming from danger android_lint. I'll create a PR for removing this suggestion.